jovian/jovian_nb_ext/main.js

```diff
@@ -638,6 +736,62 @@ define([
       jvn_params_modal.modal("show");
     };
 
+    //just save new default parameters
+    const saveParams = function() {
+      /**
+       * Initializes a dialog modal triggered by a dropdown button on the toolbar
+       *
+       * Body: formParamsUI()
+       *
+       */
+      const jvn_default_params_modal = dialog.modal({
+        show: false,
+        title: "Set Default Parameters",
+        body: formParamsUI,
+        notebook: Jupyter.notebook,
+        keyboard_manager: Jupyter.notebook.keyboard_manager,
+        buttons: {
+          Set: {
+            id: "save_params_button",
+            class: "btn-primary",
+            click: function() {
+              storeParamsInPython();
+            }
+          }
+        },
+        open: function() {
+          getParams().then(jvn_params => {
+            if (jvn_params == null) {
+              $("#nb_filename_box").val(
+                Jupyter.notebook.notebook_name.replace(".ipynb", "")
+              );
+              $($("input[name=secret_opt")[1]).prop("checked", true);
+              $($("input[name=cap_opt")[0]).prop("checked", true);
+              $($("input[name=create_opt")[1]).prop("checked", true);
+              $("#env_opt option:contains('conda')").prop("selected", true);
+            } else {
+              $("#artifacts_box").val(jvn_params.artifacts);
+              $("#files_box").val(jvn_params.files);
+              $("#notebook_id_box").val(jvn_params.notebook_id);
+              jvn_params.secret == "False"
+                ? $($("input[name=secret_opt")[1]).prop("checked", true)
+                : $($("input[name=secret_opt")[0]).prop("checked", true);
+              jvn_params.capture_env == "False"
+                ? $($("input[name=cap_opt")[1]).prop("checked", true)
+                : $($("input[name=cap_opt")[0]).prop("checked", true);
+              jvn_params.create_new == "False"
+                ? $($("input[name=create_opt")[1]).prop("checked", true)
+                : $($("input[name=create_opt")[0]).prop("checked", true);
+              jvn_params.env_type == "conda"
+                ? $("#env_opt option:contains('conda')").prop("selected", true)
```

Comment:
Storing of params is now in browser storage. Can you dicsuss with @bendesu and make this integration utilizing the current code. Check out `storeParams()` also add support to new parameters that has been added by ben.

Reply:
swapped with storeParams